open-hackathon-server/src/hackathon/hmongo/models.py

```diff
@@ -128,6 +150,11 @@ class UserToken(DynamicDocument):
     def __init__(self, **kwargs):
         super(UserToken, self).__init__(**kwargs)
 
+    # TODO: should inherit this from HDocumentBase? then should we delete the default
+    # fields in the HDocumentBase?
```

Comment:
Why should we delete the default fields? I add `create_time` and `update_time` in `HDocumentBase` on purpose.